contracts/ForkableRealityETH_ERC20.sol

```diff
@@ -215,7 +215,7 @@ contract ForkableRealityETH_ERC20 is BalanceHolder_ERC20 {
     /// @return The ID of the newly-created template, which is created sequentially.
     function createTemplate(
         string memory content
-    ) private stateAny returns (uint256) {
+    ) public stateAny returns (uint256) {
```

Comment:
This should only be used on L1 and only for governance so it shouldn't need to be public